base/system/sys_info.cc

```diff
@@ -38,6 +45,7 @@ int SysInfo::NumberOfEfficientProcessors() {
 
 // static
 uint64_t SysInfo::AmountOfPhysicalMemory() {
+#if !BUILDFLAG(USE_STARBOARD_MEDIA)
```

Comment:
Are these issues specific to using starboard media? I.e. if we build cobalt but use the chromium player, are we able to use --enable-low-end-device-mode as is?

Reply:
Chromium media pipeline (video overlay) doesn't work with low-end-device-mode.

Reply:
If that's the case, then should we be using that flag at all? It seems to me that if it doesn't work upstream for our use case, we wouldn't want to try to make it work for Cobalt, but instead look at the optimizations that flag makes and use them for Cobalt when we want